### PR TITLE
[RFR] Put some design

### PR DESCRIPTION
**Pull request by BaptistePDJ**

### CODE REVIEW

**Review by julienmilazzo — COMMENTED** [review]
myfile

File: project/templates/security/login.html.twig [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {% extends 'base.html.twig' %}
 
-{% block title %}Log in!{% endblock %}
+{% block title %}Connection{% endblock %}
```
Comment (julienmilazzo) [inline project/templates/security/login.html.twig]: connexion

File: project/templates/security/login.html.twig [hunk 2]
```diff
@@ -55,8 +43,8 @@
                     </label>
                 </div>
 
-                <button class="btn btn-lg btn-primary" type="submit">
-                    Sign in
+                <button class="btn btn-lg blue" type="submit">
+                    Connection
```
Comment (julienmilazzo) [inline project/templates/security/login.html.twig]: Connexion

File: project/templates/sidebar.html.twig [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+            </div>
+            <div class="home-admin-link">
+                <a href="{{ path('domain_name_index') }}" class="btn btn-link"><i class="fa fa-id-card" aria-hidden="true"></i> Nom de domaines</a>
+            </div>
+            <div class="home-admin-link" id="step-3">
+                <a href="{{ path('client_index') }}" class="btn btn-link"><i class="fa fa-users" aria-hidden="true"></i> Clients</a>
+            </div>
+        </div>
+    </div>
+</div>
```
Comment (BaptistePDJ) [inline project/templates/sidebar.html.twig:22]: online when your connected

**Review by julienmilazzo — APPROVED** [review]

**Review by julienmilazzo — APPROVED** [review]

**Pull request merged**